### PR TITLE
XUI update flag

### PR DESCRIPTION
**Pull request by ernestman28**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: k8s/ithc/common/xui/webapp.yaml [hunk 1]
```diff
@@ -29,8 +29,11 @@ spec:
         LAUNCH_DARKLY_CLIENT_ID: 1
         DEBUG: xuiNode:*,-xuiNode:auth:s2s
         ISS_SERVICE: https://forgerock-am.service.core-compute-idam-ithc.internal:8443/openam/oauth2/hmcts
-        SERVICES_IDAM_OAUTH_CALLBACK_URL: /oauth2/callback
+        SERVICES_IDAM_OAUTH_CALLBACK_URL: oauth2/callback
```
Comment (udaydenduluri33) [inline k8s/ithc/common/xui/webapp.yaml:32]: Why are you removing this /

Comment (ernestman28) [reply k8s/ithc/common/xui/webapp.yaml:32]: same setting as MC demo

Comment (ernestman28) [reply k8s/ithc/common/xui/webapp.yaml:32]: this is currently in MC demo which is working right, if we going to use the same tag, this env var will need to use the same too? @udaydenduluri33 https://github.com/hmcts/cnp-flux-config/blob/master/k8s/demo/common/xui/webapp.yaml#L33

Comment (udaydenduluri33) [reply k8s/ithc/common/xui/webapp.yaml:32]: If using the new node lib, then it's /ouath2/callback

Comment (ernestman28) [reply k8s/ithc/common/xui/webapp.yaml:32]: we using exact same tagged image as demo, so env var should be the same right? until we get a new PR, i think thats should be correct

**Pull request closed without merging**